pkg/kds/v2/store/sync.go

```diff
@@ -310,3 +319,11 @@ func GlobalSyncCallback(
 		},
 	}
 }
+
+func ToResourceKeys(rs core_model.ResourceList) []core_model.ResourceKey {
+	rkey := []core_model.ResourceKey{}
+	for _, r := range rs.GetItems() {
+		rkey = append(rkey, core_model.MetaToResourceKey(r.GetMeta()))
+	}
+	return rkey
+}
```

Comment:
feels like this function is generally useful no? Maybe it should be moved in `core_model` ? Maybe even a method of `ResourceList` (`GetItemsAsResourceKey()`)? Or `MetaListToResourceKeys(core_model.ResourceList)` to mirror what already exists